src/Auth0.OidcClient.WPF/PlatformWebView.cs

```diff
@@ -17,14 +17,17 @@ public class PlatformWebView : IBrowser
         /// </summary>
         /// <param name="windowFactory"> </param>
         /// <param name="shouldCloseWindow"> Determines whether the window closes or not after sucessful login</param>
-        
-        /// Example Usage:
+        /// <example> 
+        /// This sample shows how to call the <see cref="PlatformWebView(Func{Window}, bool)"/> constructor.
+        /// <code>
         /// Window ReturnWindow()
         /// {
         ///     return window; // your WPF applciation window where you want the login to pop up
```

Comment:
Typo on 'application'

Reply:
Corrected in https://github.com/auth0/auth0-oidc-client-net/pull/40/commits/652a7f5110328ceb0e23f39e4814f73856da2964